modules/protobuf/metadata.json

```diff
@@ -22,9 +22,9 @@
             "name": "Jerry Berg"
         },
         {
-            "email": "acozzette@google.com",
-            "github": "acozzette",
-            "name": "Adam Cozzette",
+            "email": "acozette@google.com",
```

Comment:
this seems wrong? :)

Reply:
fixed in the pr, and have a change to fix this in our template to not get clobbered again next time!